docs/source/configuration.rst

```diff
@@ -68,6 +70,15 @@ Values:
 - ``get_zcopy``
 - ``auto`` (default)
 
+``UCX_TCP_RX_SEG_SIZE``
+Size of send copy-out buffer when receiving.  This environment variable controls the size of the buffer on the host when receiving data over TCP.
+
+``UCX_TCP_TX_SEG_SIZE``
+
+Size of send copy-out buffer when transmitting.  This environment variable controls the size of the buffer on the host when sending data over TCP.
+
+When mixing TCP with other transports methods, users should take care to properly tune these parameters.  UCX-Py uses ``8M`` as the default value for both RX/TX.
+
```

Comment:
I would also add a note like:

Note that these variables will impact CUDA transfers as well when no NVLink or InfiniBand is available between two UCX-Py processes. The `UCX_TCP_RX_SEG_SIZE` and `UCX_TCP_TX_SEG_SIZE` will cause the `HostToDevice` and `DeviceToHost` copies of buffers to be broken down in several chunks when the size of a buffer exceeds the size defined by these two variables. If an application is expected to transfer very large buffers, increasing such values may improve overall performance.